scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -695,7 +695,8 @@ object TreeSyntax {
         }
       case t: Type.ByName => m(ParamTyp, s(kw("=>"), " ", p(Typ, t.tpe)))
       case t: Type.Var => m(SimpleTyp, s(t.name.value))
-      case t: Type.TypedParam => m(SimpleTyp, s(t.name.value), ": ", p(Typ, t.typ))
+      case t: Type.FuncParam => m(ParamTyp, w(t.mods, " "), p(Typ, t.tpe))
```

Comment:
Are there cases that cover both `TypedParam` and `FuncParam` with erased keyword?

Reply:
I meant test cases, sorry :sweat:

Reply:
actually, I think there are some removed files for https://github.com/scalameta/scalameta/actions/runs/5313174771/jobs/9618700633?pr=3189 that we could unignore.

Reply:
added test cases in SyntacticSuite. others are in ErasedDefsSuite and will change as the parser starts supporting.